### ISSUE TITLE
Make tag cloud available for public users

### ISSUE THREAD

**Issue by thelazyoxymoron** [jonschoning/espial#48]
If I'm looking at someone else's pins, I see the tags associated with individual entries, but don't see an exhaustive set of tags available for their account. Can the tag cloud be made publicly available so that any non logged-in user can also interact with it? Currently it's only visible if you're logged in.

**Comment by jonschoning** [comment]
The idea of of hiding the tag cloud for someone else's account was initially done to intentionally limit what information was exposed publicly at a glance i.e. this was someone done in the name of privacy.  

However, I can also see the argument that a user's tag cloud should be visible based on their public bookmarks. 

So there are the following options: 

1) leave the current behavior as-is
2) make the public tag cloud always visible to other users based off their public bookmarks
3) give the user an option to enable/disable their public tag cloud, with a default setting
4) set a server config option to enable/disable public tag cloud for all users, with a default setting

I'm not 100% sure which way to go but I'm leaning to 3) or 4).

**Comment by jonschoning** [comment]
The idea of of hiding the tag cloud for someone else's account was initially done to intentionally limit what information was exposed publicly at a glance i.e. this was someone done in the name of privacy.  

However, I can also see the argument that a user's tag cloud should be visible based on their public bookmarks. 

So there are the following options: 

1) leave the current behavior as-is
2) make the public tag cloud always visible to other users based off their public bookmarks
3) give the user an option to enable/disable their public tag cloud, with a default setting
4) set a server config option to enable/disable public tag cloud for all users, with a default setting

I'm not 100% sure which way to go but I'm leaning to 3) or 4).

**Comment by thelazyoxymoron** [comment]
Yes, I think option 3 sounds most reasonable from a user experience perspective.

**Comment by thelazyoxymoron** [comment]
Sorry for the bump. Just checking-in to see if this request is under consideration or not.

I would've sent a PR, but alas, no Haskell experience.

**Comment by jonschoning** [comment]
Sorry I've been a bit distracted.   I'll try to make this a higher priority at get to it soon
